specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json

```diff
@@ -52,7 +52,7 @@
           "SignalR"
         ],
         "description": "Checks that the SignalR name is valid and is not already in use.",
-        "operationId": "Signalr_CheckNameAvailability",
+        "operationId": "SignalR_CheckNameAvailability",
```

Comment:
Why don't you call the operation ID directly "Accounts"? right now, this is redundant: we know it's signal R and gives Python code like that (but I guess it will be close in other language as well):
```python
import azure.mgmt.signalr
client = azure.mgmt.signalr.SignalRManagementClient(**parameters)
client.signal_r.create_or_update(**parameters)
```
wouldn't be better with:
```python
import azure.mgmt.signalr
client = azure.mgmt.signalr.SignalRManagementClient(**parameters)
client.accounts.create_or_update(**parameters)
```

Reply:
Note that I said "accounts", but I mean "something with is not the 'signal_r' string"

Reply:
Issue #2899

Reply:
We decided to using Signalr for OperationId/Schema Id for the client SDKs.

Reply:
See PR https://github.com/Azure/azure-rest-api-specs/pull/2904. In the new version, it would look like:
```
import azure.mgmt.signalr
client = azure.mgmt.signalr.SignalrManagementClient(**parameters)
client.signalr.create_or_update(**parameters)
```

To keep the name consistent, we updated all Schema Ids such as `SignalrResource`, `SignalrManagementClient`.  I checked the generated Python/.Net SDK files, the file names and class names are expected as `Signalr`. `SignalR` will only exist in comments, descriptions and RP namespace.